### PR TITLE
CLI: Add `--exclude-checks` support

### LINKED ISSUE #308: CLI: make it easier to exclude specific checks
Right now it's only possible to pass an allowlist, not a denylist

### PR DESCRIPTION
**Pull request by swissspidy**
Fixes #308

Successfully tested using something like this:

```
wp plugin check web-stories # Returns some errors
wp plugin check web-stories --exclude-checks=late_escaping,plugin_readme,plugin_review_phpcs # No errors
```

### CODE REVIEW

File: includes/CLI/Plugin_Check_Command.php [hunk 1]
```diff
@@ -62,6 +62,10 @@ public function __construct( Plugin_Context $plugin_context ) {
 	 * [--checks=<checks>]
 	 * : Only runs checks provided as an argument in comma-separated values, e.g. i18n_usage, late_escaping. Otherwise runs all checks.
 	 *
+	 * [--exclude-checks=<checks>]
```
Comment (felixarntz) [inline includes/CLI/Plugin_Check_Command.php:65]: You call this `ignore-checks` / `ignore-slugs` everywhere else in this PR. I don't feel strongly about either way, but we should go with a consistent term throughout. It makes sense that the parameter uses "checks" while the code uses "slugs", but I mean we should only use the term "ignore" or "exclude" throughout.

I personally prefer `exclude-checks` I think, but either way WFM.

File: includes/Checker/Abstract_Check_Runner.php [hunk 2]
```diff
@@ -109,6 +117,14 @@ abstract protected function get_plugin_param();
 	 * @return array An array of Check slugs.
 	 */
 	abstract protected function get_check_slugs_param();
+	/**
```
Comment (felixarntz) [inline includes/Checker/Abstract_Check_Runner.php]: Nit-pick:
```suggestion
	abstract protected function get_check_slugs_param();

	/**
```

**Review by felixarntz — COMMENTED** [review]
@swissspidy Almost LGTM, only I think we should choose a consistent name for the new argument.

File: includes/CLI/Plugin_Check_Command.php (hunk 1 above)
Comment (swissspidy) [reply includes/CLI/Plugin_Check_Command.php:65]: Makes sense, let's settle on _exclude_

**Review by felixarntz — APPROVED** [review]
Thanks @swissspidy, LGTM!

**Comment by swissspidy** [comment]
@adamsilverstein Just as an FYI, I'll update the GitHub Action right after merging this one.

**Review by adamsilverstein — APPROVED** [review]
Fantastic!

**Review by mukeshpanchal27 — APPROVED** [review]

**Pull request merged**